driver/razerkbd_driver.h

```diff
@@ -13,163 +13,45 @@
 #ifndef __HID_RAZER_KEYBOARD_BLACKWIDOW_CHROMA_H
 #define __HID_RAZER_KEYBOARD_BLACKWIDOW_CHROMA_H
```

Comment:
If we're at it, please change these to `__HID_RAZER_KBD_H`

Reply:
I'll add a follow-up diff to this pull request to do that.

Reply:
See second patch in this pull request.